tests/unit/transformers/test_categorical.py

```diff
@@ -458,16 +458,20 @@ def test__transform_error(self):
         If the data being transformed is not in ``self.order`` an error should be raised.
         """
         # Setup
-        data = pd.Series([1, 2, 3, 2, 1, 4])
+        data_error = pd.Series([1, 2, 3, 2, 1, 4])
+        data = pd.Series([1, 2, 1, 2, 1, 1])
         transformer = OrderedUniformEncoder(order=[2, 1])
 
         # Run / Assert
+        transformer._fit(data)
```

Comment:
I think it makes sense to move the _fit/_transform into their own test and leave this one as it was, otherwise this one gets confusing.

Reply:
Yes I agree, reverted the change in [df0ad68](https://github.com/sdv-dev/RDT/pull/681/commits/df0ad68efa2fd0b78ff118934e25ef03f8a3a926).